### PR TITLE
Add Python question on Dunder methods

### PR DESCRIPTION
**Pull request by khairalanam**
## Summary of changes

The PR adds a question in the Python quiz about dunder methods.

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

### CODE REVIEW

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -1668,6 +1668,17 @@ const pythonQuiz = [
     Explanation:
       "This is a list comprehension that prints a list of squares from 0 to 9. We use list comprehensions to simplify one-liner operations in a for loop",
     Link: "https://www.freecodecamp.org/news/list-comprehension-in-python-with-code-examples/"
+  },
+  {
+    Question:
+      "Which of the following dunder methods allows you to do the expression x + y, where x and y are two objects of the same class?",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1674]:
```suggestion
      "In Python, which of the following dunder methods defines the behavior of the addition operator + for objects of a class?",
```

File: src/data/python-quiz.ts [hunk 2]
```diff
@@ -1668,6 +1668,17 @@ const pythonQuiz = [
...
+    Answer: "__add__",
+    Distractor1: "__len__",
+    Distractor2: "__sub__",
+    Distractor3: "__mul__",
+    Explanation:
+      "Dunder methods (Double underscore) are special methods reserved in Python which affect's the behaviour of the object and the interaction of the object with other objects.",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:1680]:
```suggestion
      "Dunder methods (Double underscore) are special methods reserved in Python which affect's the behavior of the object and the interaction of the object with other objects.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by khairalanam** [comment]
I have implemented the changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by khairalanam** [comment]
Thank you!

**Pull request merged**